index.js

```diff
@@ -24,7 +24,11 @@ function Gas (runner, options) {
   // Load config / keep .ethgas.js for backward compatibility
   let config;
   if (options && options.reporterOptions){
-    config = options.reporterOptions
+    const argv = require('minimist')(process.argv.slice(2))
+    config = {
+      ...options.reporterOptions,
+      ...argv
+    }
```

Comment:
This is actually causing CI to crash (please look at the Travis run). 

Tool currently supports down to Node 7 which doesn't let you expand objects with the `...` operator. Additionally, if you'd like to add this command line functionality, would it be possible to do it in a separate PR which includes tests for it?

Reply:
Apologies - I guess we're supporting Node 8 (according to the README) - I'm merging a change to CI for that right now.